src/orderedmap.h

```diff
@@ -427,7 +464,11 @@ bool OrderedMap<Key, Value>::isEmpty() const
 template<typename Key, typename Value>
 QList<Key> OrderedMap<Key, Value>::keys() const
 {
-    return QList<Key>::fromStdList(insertOrder.toStdList());
+#if (QT_VERSION >= 0x050E00)
```

Comment:
```suggestion
#if (QT_VERSION >= QT_VERSION_CHECK(5, 15, 0))
```

Reply:
Can't do that as it breaks before Qt 5 - and upstream hasn't abandoned that yet...
See https://github.com/mandeepsandhu/qt-ordered-map/issues/19 and https://github.com/mandeepsandhu/qt-ordered-map/pull/20 .

Reply:
Good call.